### PR TITLE
Add send transaction types

### PR DESCRIPTION
**Pull request by rufflian**
- Export for use by program implementations

### CODE REVIEW

File: src/transaction/send.ts [hunk 1]
```diff
@@ -344,4 +344,9 @@ async function awaitTransactionSignatureConfirmation(
   done = true;
   log.debug("Returning status", status);
   return status;
+}
+
+export interface SentTransactionDetails {
```
Comment (long-banana) [inline src/transaction/send.ts]: Let’s call this SendTransactionResult. Ties the naming closer to the return of the function call vs. details about the sent transaction. If we end up adding more information in the return that might not just be related to the transaction itself, the name will still fit.

**Review by long-banana — APPROVED** [review]
Address the one comment I left and you’re free to merge yourself. 👍🏻

**Comment by rufflian** [comment]
> Address the one comment I left and you’re free to merge yourself. 👍🏻

@long-banana I don't have write access 🤷 
Screenshot 2022-07-20 165147

File: src/transaction/send.ts [hunk 2]
```diff
@@ -344,4 +344,9 @@ async function awaitTransactionSignatureConfirmation(
   done = true;
   log.debug("Returning status", status);
   return status;
+}
+
+export interface SentTransactionResult {
```
Comment (long-banana) [inline src/transaction/send.ts]: SendTransactionResult please. Same tense as the function call name

Comment (rufflian) [reply src/transaction/send.ts]: My bad! Updated

**Pull request merged**